docs/About.md

```diff
@@ -2,6 +2,19 @@
 
 # Field Mapping Tasking Manager (FMTM)
 
+## :open_book: History 
+_How was FMTM born ?_
+<br>
+<br>
+It started as Ivan's idea to build FMTM (Ivan Gayton is Senior Humanitarian Advisor at Humanitarian OpenStreetMap Team) which then became a collaborative project with the efforts of Ivan , Rob Savoye who is Senior Technical Lead at Humanitarian OpenStreetMap Team and many other members from HOT. 
```

Comment:
@neelimagoogly  please add  after "many other members from HOT as well as volunteers interested in the project".

Reply:
Sure 👍